lib/child_process.js

```diff
@@ -565,6 +565,12 @@ function normalizeSpawnArguments(file, args, options) {
 }
 
 
+function escapeArgument(str) {
+  if (str === '') return '\'\'';
+  if (!/[^%+,-./:=@_0-9A-Za-z]/.test(str)) return str;
```

Comment:
Where did this list come from? Is any of this platform-specific?

Reply:
@mscdex -This contain all the characters that are allowed in a windows and unix-shell, without needing an escaping.

Reply:
You can use primordials here:
```suggestion
  if (!RegExpPrototypeTest(/[^%+,-./:=@_0-9A-Za-z]/, str)) return str;
```

Reply:
fixed.